.eslintignore

```diff
@@ -0,0 +1 @@
+src/scripts/modules
```

Comment:
@Maks-s can you check if it is working?

After running the following with this patch I do not get any occurrence of `script/module` anymore.

```
$ DEBUG=eslint:cli-engine eslint -c .eslintrc.js --fix ./src/scripts/
```

Cheers,

Reply:
Working perfectly, but I don't understand why would you skip `script/module` 🤔

Reply:
Me either :thinking: 

I just assume that it was the excepted behavior so we have to make the excepted behavior work.

Let's ping @daed because he initiated that particular line with https://github.com/encryptic-team/encryptic/commit/e4b23f134e55a5a30fcd9407507b6f82ae6f58ba .

Reply:
Oh ok, I think it's ignored because in the original laverna repo they wanted to have a clean eslint with no warnings : 
https://github.com/Laverna/laverna/commit/15f197d627fe3ab5f84108be82abbabcc495170c#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

Reply:
It was actually introduced by https://github.com/Laverna/laverna/commit/a2c2922078ad92d8b337863839b3070900dbcca1 :thinking:

Reply:
Nah, Laverna/laverna@a2c2922 only remove `--ignore-pattern='encryption'` but `--ignore-pattern 'app/scripts/modules` is already here